parser/parser_test.go

```diff
@@ -342,12 +349,24 @@ world`},
 				Left:  &IdentifierNode{Value: "foo"},
 				Right: &StringNode{Value: "foo"}},
 		},
+		{
+			`startsWith(foo,  "foo")`,
+			&BinaryNode{Operator: "startsWith",
+				Left:  &IdentifierNode{Value: "foo"},
+				Right: &StringNode{Value: "foo"}},
+		},
 		{
 			`foo endsWith "foo"`,
 			&BinaryNode{Operator: "endsWith",
 				Left:  &IdentifierNode{Value: "foo"},
 				Right: &StringNode{Value: "foo"}},
 		},
+		{
+			`endsWith(foo,  "foo")`,
+			&BinaryNode{Operator: "endsWith",
+				Left:  &IdentifierNode{Value: "foo"},
+				Right: &StringNode{Value: "foo"}},
+		},
```

Comment:
And lets also add a test withing expr_test to see if it is possible to add own function there?